src/Mail/SendmailMailer.php

```diff
@@ -53,7 +53,7 @@ public function send(Message $mail): void
 			str_replace(Message::EOL, PHP_EOL, (string) $mail->getEncodedHeader('To')),
 			str_replace(Message::EOL, PHP_EOL, (string) $mail->getEncodedHeader('Subject')),
 			str_replace(Message::EOL, PHP_EOL, $parts[1]),
-			str_replace(Message::EOL, PHP_EOL, $parts[0]),
+			str_replace(Message::EOL, PHP_VERSION_ID >= 80000 ? "\r\n" : PHP_EOL, $parts[0]),
```

Comment:
Does this also apply to other parameters? Then it would probably be best to remove the workaround and use:

```php
		$args = [
			(string) $mail->getEncodedHeader('To'),
			(string) $mail->getEncodedHeader('Subject'),
			$parts[1],
			$parts[0],
		];
```

Reply:
As far as I know, this issue it's related only to `$additional_headers`.